internal/systemd/systemd.go

```diff
@@ -0,0 +1,108 @@
+package systemd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Systemd interface {
+	Enable(ctx context.Context, service string) error
+	Disable(ctx context.Context, service string) error
+	IsEnabled(ctx context.Context, service string) (bool, error)
+	Close()
+}
+
+type SystemdConnector struct {
+	dbusConnection DbusConnector
+	logger         *logrus.Entry
```

Comment:
Let's catch up with this: https://github.com/trento-project/workbench/pull/25
And use `"log/slog"`.
You'll find examples in the PR